tests/conftest.py

```diff
@@ -798,6 +798,7 @@ def design_workflow_dict(generic_entity):
     ret = generic_entity.copy()
     ret.update({
         "name": "Example Design Workflow",
+        "description": "A description! Of the Design Workflow! So you know what it's for!",
```

Comment:
Did you mean to include `description` twice?

Reply:
noooope 🤣

Reply:
this still needs to be removed